ensime-sbt.el

```diff
@@ -144,14 +151,13 @@ again."
                                       (?q "test-[q]uick" "testQuick"))))
     (concat module "/" source-set task)))
 
-(defun ensime-sbt-find-subproject (file-name source-set)
-  (when source-set
-    (let* ((config (-> (ensime-connection) ensime-config))
-           (subprojects (plist-get config :subprojects))
-           (matches-subproject-dir? (lambda (dir) (string-match-p dir file-name)))
-           (find-subproject (lambda (sp)
-                              (-any matches-subproject-dir? (plist-get sp :source-roots)))))
-      (-> (-find find-subproject subprojects) (plist-get :name)))))
+(defun ensime-sbt-find-subproject (file-name)
+  (let* ((config (-> (ensime-connection) ensime-config))
+         (subprojects (plist-get config :subprojects))
+         (matches-subproject-dir? (lambda (dir) (string-match-p dir file-name)))
```

Comment:
heh, I think my lisp foo has increased since this code was merged... I would definitely want to see functions defined with `cl-flet` now, not a scheme-style `let`.